libs/community/langchain_community/vectorstores/milvus.py

```diff
@@ -989,3 +989,69 @@ def _parse_document(self, data: dict) -> Document:
             page_content=data.pop(self._text_field),
             metadata=data.pop(self._metadata_field) if self._metadata_field else data,
         )
+
+    def get_ids(self,
```

Comment:
Minor point - I would change this to `get_pks` to avoid confusion.

Reply:
Hi @fzliu ,
Yup. I totally agree. So I have changed it by moving `get_ids` to `get_pks`